src/main/java/controller/MainPageServlet.java

```diff
@@ -0,0 +1,37 @@
+package controller;
+
+import factory.UserServiceFactory;
+import org.apache.log4j.Logger;
+import service.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/authorization")
+public class MainPageServlet extends HttpServlet {
+    private static final Logger logger = Logger.getLogger(MainPageServlet.class);
+    private static final UserService USER_SERVICE = UserServiceFactory.getUserService();
```

Comment:
```suggestion
    private static final UserService userService = UserServiceFactory.getUserService();
```
in this situation we need to write name in camelCase, because this field is not constant. It is is mutable and haven't some default data